scripts/functions/help_system

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+function print_arrays()
+{
+    #### This function uses indirect expansion (pass variable name, not the value)
+    ##
+    ## Syntax:  print_arrays args args_about
+    ##
+    ## Where args and args_about are arrays containing double quoted content:
+    ##
+    ##   args=("-h|--help"\
+    ##         "-a|--absolute [INT]"\
+    ##         "-b|--bool-option")
+    ##
+    ##   args_about=("Pring this message and exit"\
+    ##               "-a does this"\
+    ##               "-b does that")
+    ##
+    ## Note: Line Continuations "\" are _required_ when building multi-argument help systems
+    ####
+
+    ## The desired maximium about text length, before justification occurs
+    local max_length=60
```

Comment:
that's pretty short. we picked 132 in moose due to being a gnome preset

Reply:
Hmm, I will see how that looks. 60 is pretty sexy as-is IMHO. And is what makes that example I left in comments about the syntax look the way it does. Also, I believe this is 60 characters of argument _description_ and not overall length.